src/pages/find-my-combos.tsx

```diff
@@ -117,6 +121,22 @@ const FindMyCombos = () => {
     setDecklist(e.target.value);
   };
 
+  const handleUrlInput = async (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
While the expected use case is that a user will paste in a url, it is always possible that they might type it in instead (however strange). The way it currently is, we'd be sending a fetch request every single key stroke as they completed a valid url. 

I'd make a separate onChange handling and then add a submit button that handles this function onClick.